src/open_r1/rewards.py

```diff
@@ -368,13 +383,13 @@ def extract_code(completion: str, language: str = "python") -> str:
     return extracted_answer
 
 
-def binary_code_reward(completions, **kwargs) -> list[float]:
-    rewards = code_reward(completions, **kwargs)
+def binary_code_reward(completions, num_parallel: int = 2, **kwargs) -> list[float]:
+    rewards = code_reward(completions, num_parallel=num_parallel, **kwargs)
     BINARY_THRESHOLD = 0.99
```

Comment:
Not related to this PR, but why do we need this threshold? Aren't the rewards always 1 or 0 in this case?

Reply:
The rewards are in the range 0-1 depending on how many unit tests pass. I use a threshold as we may want to vary it in the future, but I did not want to expose it yet until we feel the need.

Reply:
Ah I see, in that case wouldn't a threshold of 0.5 be more natural (as one does with e.g. classifiers)? Can naturally be left as something to follow-up on if we're not getting enough signal with the current threshold